packages/contentful/src/crn/queries/users.queries.ts

```diff
@@ -57,6 +57,17 @@ export const usersContentQueryFragment = gql`
           }
           displayName
           inactiveSince
+          linkedFrom {
+            interestGroupsCollection {
```

Comment:
I think the default limit is 100 anyway, but for consistency with working groups.

```suggestion
            interestGroupsCollection(limit: 100) {
```